core/trino-spi/src/main/java/io/trino/spi/type/TypeOperators.java

```diff
@@ -199,8 +199,7 @@ public synchronized MethodHandle get()
         private MethodHandle adaptOperator(OperatorConvention operatorConvention)
         {
             OperatorMethodHandle operatorMethodHandle = selectOperatorMethodHandleToAdapt(operatorConvention);
-            MethodHandle methodHandle = adaptOperator(operatorConvention, operatorMethodHandle);
-            return methodHandle;
+            return adaptOperator(operatorConvention, operatorMethodHandle);
```

Comment:
In some places i think the non-inlined version is used because it helps with putting a breakpoint and seeing the returned value. Not sure if it was intentional here.

Reply:
CC: @dain (This code seems to have been added 2 years ago)

Reply:
Reverted.